core/templates/dev/head/pages/skill_editor/editor_tab/add_misconception_modal_directive.html

```diff
@@ -33,22 +33,22 @@ <h3>Add Misconception</h3>
         <div>
           <span class="oppia-editor-description">These are additional notes for question creators to understand how handling this misconception is useful for the skill being tested.</span>
         </div>
-        <schema-based-editor schema="MISCONCEPTION_PROPERTY_FORM_SCHEMA" local-value="misconceptionNotes">
+        <schema-based-editor class="protractor-test-notes-textarea" schema="MISCONCEPTION_PROPERTY_FORM_SCHEMA" local-value="misconceptionNotes">
         </schema-based-editor>
       </div>
       <div class="oppia-misconception-editor-section">
         <strong>Default Feedback</strong>
         <div>
           <span class="oppia-editor-description">This is the default feedback given to learners when they have this misconception. Question creators may override this feedback.</span>
         </div>
-        <schema-based-editor schema="MISCONCEPTION_FEEDBACK_PROPERTY_FORM_SCHEMA" local-value="misconceptionFeedback">
+        <schema-based-editor class="protractor-test-feedback-textarea" schema="MISCONCEPTION_FEEDBACK_PROPERTY_FORM_SCHEMA" local-value="misconceptionFeedback">
         </schema-based-editor>
       </div>
     </div>
   </div>
 
   <div class="modal-footer">
     <button class="btn btn-default" ng-click="cancel()">Cancel</button>
-    <button class="btn btn-success" ng-click="saveMisconception()" ng-disabled="!misconceptionName || misconceptionNotes.length === 0 || misconceptionFeedback.length === 0">Save</button>
+    <button class="btn btn-success protractor-test-confirm-add-misconception" ng-click="saveMisconception()" ng-disabled="!misconceptionName || misconceptionNotes.length === 0 || misconceptionFeedback.length === 0">Save</button>
```

Comment:
`protractor-test-confirm-add-misconception-button` Add '-button' to the class names of buttons.